src/trialsynth/base/ground.py

```diff
@@ -0,0 +1,75 @@
+import gilda
+import copy
+
+from typing import Callable, Iterator, Optional
+from indra.databases import mesh_client
+
+from .models import BioEntity
+
+Grounder = Callable[[BioEntity, list[str], str], Iterator[BioEntity]]
+
+condition_namespaces = ['MESH', 'DOID', 'EFO', 'HP', 'GO']
+intervention_namespaces = ['CHEBI', 'MESH', 'EFO', 'HGNC']
+
+PreProcessor = Callable[[BioEntity], BioEntity]
+
+
+def ground_entity(
+        entity: BioEntity,
+        preprocessor: PreProcessor,
```

Comment:
This could have a default None here and below, we set to preprocessor = lambda x: x